src/validators/callout-component-content.validator.ts

```diff
@@ -0,0 +1,13 @@
+import { isCalloutContentComponent } from "../utils/index.ts";
+import type { CalloutComponentSchema } from "../types/index.ts";
+
+export const calloutComponentContentValidator = (
+  schema: CalloutComponentSchema,
```

Comment:
I think for all of these validators you can enforce the schema type rather than checking it at runtime, i.e. the validator just needs to validate the `answer`.

```ts
export const calloutComponentContentValidator = (
  schema: CalloutComponentContentSchema,
  answer: unknown,
): boolean => {
```